VDF.GUI/Views/MainWindow.xaml

```diff
@@ -511,10 +512,34 @@
                                             </DataTemplate>
                                         </DataGridTemplateColumn.CellTemplate>
                                     </DataGridTemplateColumn>
-                                    <DataGridTextColumn
-                                        Width="1*"
-                                        Binding="{Binding ItemInfo.Similarity}"
-                                        Header="Similarity" />
+                                    <DataGridTemplateColumn Width="1*">
+                                        <DataGridTemplateColumn.Header>
+                                            <StackPanel>
+                                                <TextBlock Text="Similarity" />
+                                            </StackPanel>
+                                        </DataGridTemplateColumn.Header>
+                                         <DataGridTemplateColumn.CellTemplate>
+                                            <DataTemplate>
+                                                <Grid Margin="10,0,0,0">
+                                                    <Grid.RowDefinitions>
+                                                        <RowDefinition />
+                                                        <RowDefinition />
+                                                        <RowDefinition />
+                                                    </Grid.RowDefinitions>
+                                                    <TextBlock
+                                                        Grid.Row="1"
+                                                        Margin="2"
+                                                        Text="{Binding ItemInfo.Similarity, StringFormat=\{0:0.##\}}" />
+                                                    <TextBlock
+                                                        Grid.Row="2"
+                                                        Margin="2"
+                                                        IsVisible="{Binding ItemInfo.Flags, Converter={StaticResource IsFlaggedConverter}}"
+                                                        Text="{Binding ItemInfo.Flags}"
+                                                        TextWrapping="Wrap" />
+                                                </Grid>
+                                            </DataTemplate>
+                                        </DataGridTemplateColumn.CellTemplate>
+                                    </DataGridTemplateColumn >
```

Comment:
I don't think this is necessary. The preview image already reveals if its a flipped match or not.

Reply:
OK (I'll interpret it as: Please revert the change.)

- Is it ok to keep `StringFormat=\{0:0.##\}` or did you intentionally display so many decimal places?
- If not showing the flipped flag, ItemInfo.Flags has in principle no more use at this moment. So does your request also includes the removal of the flag at all?

_My thoughts on this were that the specification of the percentage similarity on its own no longer has the full significance. Or that one recognizes with many duplicate findings the flipped property then also more simply,...
The flag was also only displayed if it is != None in order not to disturb in the normal case.
But of course the basic statement is right, that if you look at the pictures yourself, you should recognize the flipping in the rule._